sys-libs/libxcrypt/libxcrypt-4.4.19.ebuild

```diff
@@ -52,19 +66,50 @@ src_configure() {
 	multibuild_foreach_variant multilib-minimal_src_configure
 }
 
+get_xcsysroot() {
+	if is_cross; then
+		echo "${EPREFIX}/usr/${CTARGET}"
+	else
+		echo "/"
+	fi
+}
+
 get_xclibdir() {
-	printf -- "%s/%s/%s\n" \
+	printf -- "%s/%s/%s/%s\n" \
+		"$(get_xcsysroot)" \
 		"$(usex split-usr '' '/usr')" \
 		"$(get_libdir)" \
 		"$(usex system '' 'xcrypt')"
 }
 
+get_xcincludedir() {
+	printf -- "%s/%s/include/%s\n" \
+		"$(get_xcsysroot)" \
+		"$(usex split-usr '' '/usr')" \
+		"$(usex system '' 'xcrypt')"
+}
+
+get_xcmandir() {
+	printf -- "%s/%s/share/man\n" \
+		"$(get_xcsysroot)" \
+		"$(usex split-usr '' '/usr')"
+}
+
+get_xcpkgconfigdir() {
+	printf -- "%s/%s/%s/pkgconfig\n" \
+		"$(get_xcsysroot)" \
+		"$(usex split-usr '' '/usr')" \
+		"$(get_libdir)"
+}
+
 multilib_src_configure() {
```

Comment:
As I mentioned, I think we can get rid of most/all of these helpers above by using the right variables provided in EAPI 7

Reply:
Will do.